src/components/pages/InstructorNewsFeed/NewsFeedPutModal.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState,useEffect } from 'react';
+import { Form, Input, Button } from 'antd';
+import axios from 'axios';
+import '../../../styles/InstructorStyles/index.less';
+
+export default function NewsfeedPutModal(props) {
+  const { setPostOptions, postID } = props;
+  const token = JSON.parse(localStorage.getItem('okta-token-storage'));
+  const config = {
+    headers: { Authorization: `Bearer ${token.idToken.value}` },
+  };
+  const [formValue, setformValue] = useState({
+    link: '',
+    description: '',
+    title: '',
+  });
+  
+  useEffect(()=>{
+    axios
+      .get(`https://coder-heroes-api.herokuapp.com/news/${postID}`, config)
+      .then(resp => {
+        setformValue({
+          ...formValue,
+          link:resp.data.link,
+          description:resp.data.description,
+          title: resp.data.title
+        });
+      })
+      .catch(err => {
+        console.log('error fetching newsfeed');
+      });
+  },[]);
+  const handleChange = e => {
+    setformValue({
+      ...formValue,
+      [e.target.name]: e.target.value,
+    });
+  };
```

Comment:
I've seen these past 20-30ish lines repeated in your code, maybe some kind of global state implementation in this situation could help make your code more DRY?